wordle-web/components/settings-dialog.vue

```diff
@@ -1,74 +1,119 @@
 <template>
-  <div>
-    <v-btn icon @click="toggleDialog">
-      <v-icon> mdi-cog </v-icon>
-    </v-btn>
-
-    <v-dialog v-model="dialog" width="450">
-      <v-card>
-        <v-container>
-          <v-card-title> Settings </v-card-title>
-
-          <v-card-text>
-            <v-menu offset-y>
-              <template #activator="{ on, attrs }">
-                <v-btn color="primary" dark v-bind="attrs" v-on="on">
-                  Pick Theme
-                </v-btn>
-              </template>
-              <v-list>
-                <v-list-item-group>
-                  <v-list-item @click="turnOnTheLights">
-                    <v-list-item-title> Light Mode </v-list-item-title>
-                  </v-list-item>
-                  <v-list-item @click="turnOffTheLights">
-                    <v-list-item-title> Dark Mode </v-list-item-title>
-                  </v-list-item>
-                </v-list-item-group>
-              </v-list>
-            </v-menu>
-
-            <v-menu offset-y>
-              <template #activator="{ on, attrs }">
-                <v-btn color="primary" dark v-bind="attrs" v-on="on">
-                  Pick A Color
-                </v-btn>
-              </template>
-              <v-list>
-                <v-list-item-group>
-                  <v-list-item @click="purpleTheme">
-                    <v-list-item-title> Purple </v-list-item-title>
-                  </v-list-item>
-                </v-list-item-group>
-              </v-list>
-            </v-menu>
-          </v-card-text>
-        </v-container>
-      </v-card>
-    </v-dialog>
+    <div>      
+        <v-btn
+          icon @click ="toggleMenu">
+          <v-icon> mdi-hamburger</v-icon>
+        </v-btn>
```

Comment:
I would suggest putting this in it's own component, outside of the settings dialog.